third_party/chromium/media/BUILD.gn

```diff
@@ -248,6 +248,7 @@ component("media") {
     "//build:chromeos_buildflags",
     "//net",
     "//starboard:starboard_group",
+    "//third_party/libvpx",
```

Comment:
I believe libvpx lives below starboard (but please correct me if I'm wrong about that), so I don't like this dependency being here. If the issue if with gn check complaining about headers being included without a dependency on the target, send me a message and we'll figure it out

Reply:
Talked offline, added a line in supported_types.cc to skip the GN check.